holidays/registry.py

```diff
@@ -339,35 +339,35 @@ def get_entity(self) -> Optional[HolidayBase]:
     @staticmethod
     def _get_entity_codes(
         container: RegistryDict,
-        entity_length: Union[int, Iterable[int]],
         include_aliases: bool = True,
     ) -> Iterable[str]:
-        entity_length = {entity_length} if isinstance(entity_length, int) else set(entity_length)
         for entities in container.values():
-            for entity in entities:
-                if len(entity) in entity_length:
-                    yield entity
-                    # Assuming that the alpha-2 code goes first.
-                    if not include_aliases:
-                        break
+            for entity in entities[1:]:
+                yield entity
+
+                # Assuming that the alpha-2 code goes first.
+                if not include_aliases:
+                    break
 
     @staticmethod
     def get_country_codes(include_aliases: bool = True) -> Iterable[str]:
         """Get supported country codes.
 
         :param include_aliases:
-            Whether to include entity aliases (e.g. UK for GB).
+            Whether to include entity aliases (e.g. GBR and UK for GB,
+            UKR for UA, USA for US, etc).
         """
-        return EntityLoader._get_entity_codes(COUNTRIES, 2, include_aliases)
+        return EntityLoader._get_entity_codes(COUNTRIES, include_aliases)
 
     @staticmethod
     def get_financial_codes(include_aliases: bool = True) -> Iterable[str]:
         """Get supported financial codes.
 
         :param include_aliases:
-            Whether to include entity aliases(e.g. TAR for ECB, XNYS for NYSE).
+            Whether to include entity aliases (e.g. B3 for BVMF,
+            TAR for ECB, XNYS for NYSE, etc).
```

Comment:
```suggestion
            TAR for ECB, NYSE for XNYS, etc).
```
XNYS is main code now (ISO 10383). I forgot to update this here and in [utils.py](https://github.com/vacanza/holidays/blob/a459edc7c19dd7adbdf3e41b8e7cd316f9a0db94/holidays/utils.py#L382).

Reply:
Updated.